### PR TITLE
3670 only show judge training under certain conditions

### PR DESCRIPTION
**Pull request by viviancan**
Refs #3670 

This change includes new css additions for use of a custom tooltip.
There are 2 main new classes `has-tooltip` and `tooltip-text`. `has-tooltip` is used as a wrapper and `tooltip-text` contains the actual text of the tooltip.

CleanShot 2022-10-17 at 17 37 46

### CODE REVIEW

File: app/javascript/packs/application_rebrand.js [hunk 1]
```diff
@@ -1,15 +1,16 @@
-import "../stylesheets/tailwind.css"
+import "../stylesheets/tailwind.css";
```
Comment (viviancan) [inline app/javascript/packs/application_rebrand.js:1]: My linter automatically added the semicolons! I tried removing them, but every time I save they are automatically added before I am able to commit. Not sure if this is part of our standard. I can look at disabling this setting if this is an issue!

File: app/views/judge/dashboards/rebrand/_side_nav_content.html.erb [hunk 2]
```diff
@@ -26,11 +26,18 @@
   <% else %>
     <%= web_icon("circle-o") %>
   <% end  %>
-  <%= link_to "Judge Training",
-              "https://technovationchallenge.org/judge-training/",
-              target: "_blank",
-              id: "judge-training-tab"
-  %>
+
+  <% if SeasonToggles.judging_enabled? || SeasonToggles.judge_registration_open? %>
```
Comment (viviancan) [inline app/views/judge/dashboards/rebrand/_side_nav_content.html.erb:30]: I researched a few libraries, but they all seemed heavier than what we needed. So I decided to go with a basic CSS custom route. 

I also tried a variety of approaches with the `link_to`method including the `link_to_if` using a block to dynamically display the tooltip span. I had a working version, but it was so dense with all of the conditional logic and extra spans, that it just didn't seem very clear in the end.

File: app/javascript/packs/application_rebrand.js (hunk 1 above)
Comment (shaunxp20) [reply app/javascript/packs/application_rebrand.js:1]: Hmm, not sure off the top of my head, but I'm guessing it's a default from ESLint or Prettier. But anywho, yeah, we should probably get specific on our rules and what we want, and make them explicit! 😃

**Review by shaunxp20 — APPROVED** [review]
Didn't try it locally, but LGTM, thanks Vivian! 👍 🙌 :shipit:

**Pull request merged**